modules/publicize/publicize.php

```diff
@@ -318,8 +318,13 @@ function save_meta( $post_id, $post ) {
 		// Did this request happen via wp-admin?
 		$from_web = 'post' == strtolower( $_SERVER['REQUEST_METHOD'] ) && isset( $_POST[$this->ADMIN_PAGE] );
 
-		if ( ( $from_web || defined( 'POST_BY_EMAIL' ) ) && !empty( $_POST['wpas_title'] ) )
-			update_post_meta( $post_id, $this->POST_MESS, trim( stripslashes( $_POST['wpas_title'] ) ) );
+		if ( ( $from_web || defined( 'POST_BY_EMAIL' ) ) && isset( $_POST['wpas_title'] ) ) { 
+ 	            if ( empty( $_POST['wpas_title'] ) ) { 
```

Comment:
technically a bunch of whitespace would be an acceptable publicize message, so I'm not keen on `trim`ing. I am not sure what `stripslashes` would accomplish in this context.